src/EXTRA-FIX/fix_wall_flow.cpp

```diff
@@ -0,0 +1,294 @@
+// clang-format off
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing authors: Vladislav Galigerov (HSE),
+                         Daniil Pavlov (MIPT)
+------------------------------------------------------------------------- */
+
+#include "fix_wall_flow.h"
+#include "atom.h"
+#include "citeme.h"
+#include "comm.h"
+#include "domain.h"
+#include "error.h"
+#include "input.h"
+#include "lattice.h"
+#include "modify.h"
+#include "update.h"
+#include "variable.h"
+#include "random_mars.h"
+#include "memory.h"
+#include "force.h"
+#include "math_const.h"
+
+#include <cstring>
+#include <iostream>
+#include <algorithm>
+#include <functional>
+#include <fstream>
+
+using namespace LAMMPS_NS;
+using namespace FixConst;
+
+/* ---------------------------------------------------------------------- */
+
+static const char cite_fix_wall_flow_c[] =
+  "fix wall/flow command: doi:10.1177/10943420231213013\n\n"
+  "@Article{Pavlov-etal-IJHPCA-2024,\n"
+  " author = {Daniil Pavlov and Vladislav Galigerov and Daniil Kolotinskii and Vsevolod Nikolskiy and Vladimir Stegailov},\n"
+  " title = {GPU-based molecular dynamics of fluid flows: Reaching for turbulence},\n"
+  " journal = {The International Journal of High Performance Computing Applications},\n"
+  " year =    2024,\n"
+  " volume =  38,\n"
+  " number =  1,\n"
+  " pages =   34-49\n"
+  "}\n\n";
+
+FixWallFlow::FixWallFlow(LAMMPS *lmp, int narg, char **arg) :
```

Comment:
In general, we prefer contributed code to follow the LAMMPS programming style for formatting, indentation, so on. This is closely reflected in the provided configuration for the "clang-format" tool. By removing the comment at the top and then running clang-format on the source, it would be reformatted to follow the expected style.
https://docs.lammps.org/Modify_style.html

Reply:
Done